lib/wagn/generators/wagn/templates/Rakefile

```diff
@@ -3,6 +3,10 @@
 # for example lib/tasks/capistrano.rake, and they will automatically be available to Rake.
 
 require File.expand_path('../config/application', __FILE__)
+
+<% if options['mod-dev'] || options['core-dev'] -%>
 require 'jasmine'
+<% end -%>
+
```

Comment:
the jasmine gem is only in mod-dev and core-dev Gemfiles.  Rake tasks stopped working with this.